packages/blob/src/index.ts

```diff
@@ -115,18 +107,30 @@ export async function put(
     }
   }
 
-  const blobResult = (await blobApiResponse.json()) as BlobMetadataApi;
-  return mapBlobResult(blobResult);
+  const blobResult = (await blobApiResponse.json()) as PutBlobApiResponse;
+
+  return blobResult;
 }
 
-type BlobDelResult<T extends string | string[]> = T extends string
-  ? BlobResult | null
-  : (BlobResult | null)[];
+// vercelBlob.del()
 
-export async function del<T extends string | string[]>(
```

Comment:
I moved away from generics to overloads because the method doesn't accepts generics.

Reply:
I don't understand the exact issue but I'm fine with the overloads too